packages/animations/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Fixes Swift format issues in example app.
```

Comment:
Optional nit: you could revert all of the CHANGELOG changes, as this would fall under the "only affect[s] development of the package" exemption; formatting of the boilerplate runner code isn't going to be of interest to clients.